src/Stl/Generators/RandomInt64Generator.cs

```diff
@@ -19,4 +20,21 @@ public override long Next()
             return _buffer![0];
         }
     }
+#else
+    public sealed class RandomInt64Generator : Generator<long>
```

Comment:
Seems like this one is equally good, so maybe keep it as the only one?

Reply:
Done